sdk/cognitivelanguage/Azure.AI.Language.Text/samples/Sample10_AnalyzeTextOperationAsync_CustomMultiLabelClassificationOperationAction.md

```diff
@@ -0,0 +1,93 @@
+
+This sample demonstrates how to perform custom multi-label classification one or more documents. In order to use this feature, you need to train a model with your own data. For more information on how to do the training, see [train model][train_model].
+
+## Create a `TextAnalysisClient`
+
+To create a new `TextAnalysisClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create with an API key.
+
+```C# Snippet:CreateTextClient
+Uri endpoint = new Uri("<your endpoint>");
+AzureKeyCredential credential = new("your apikey");
+TextAnalysisClient client = new TextAnalysisClient(endpoint, credential);;
+```
+
+The values of the `endpoint` and `apiKey` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+
+## Perform custom multi-label classification on one or more text documents
+
+To perform custom multi-label classification one or more text documents, call `AnalyzeTextOperationAsync` on the `TextAnalysisClient` by passing the documents as `MultiLanguageTextInput` parameter and a `AnalyzeTextOperationAction` with a `CustomMltiLabelClassificationOperationAction` action. This returns a `Response<AnalyzeTextOperationState>` which you can extract the `CustomMultiLabelClassificationOperationResult`.
+
+```C# Snippet:Sample10_AnalyzeTextOperationAsync_CustomMultiLabelClassificationOperationAction
+string documentA =
+    "I need a reservation for an indoor restaurant in China. Please don't stop the music. Play music and"
+    + " add it to my playlist.";
+
+// Prepare the input of the text analysis operation. You can add multiple documents to this list and
+// perform the same operation on all of them simultaneously.
+MultiLanguageTextInput multiLanguageTextInput = new MultiLanguageTextInput()
+{
+    Documents =
+    {
+        new MultiLanguageInput("A", documentA)
+        {
+            Language = "en"
+        },
+    }
+};
+
+// Specify the project and deployment names of the desired custom model. To train your own custom model to
+// recognize custom entities, see https://aka.ms/azsdk/textanalytics/customentityrecognition.
+string projectName = TestEnvironment.CMCProjectName;
+string deploymentName = TestEnvironment.CMCDeploymentName;
+
+CustomMultiLabelClassificationActionContent customMultiLabelClassificationActionContent = new CustomMultiLabelClassificationActionContent(projectName, deploymentName);
+
+var analyzeTextOperationActions = new AnalyzeTextOperationAction[]
+{
+    new CustomMultiLabelClassificationOperationAction
+    {
+        Name = "CMCOperationActionSample",
```

Comment:
Might be good to add a comment here around this being a "string for humans" and that it doesn't need to be carefully selected